test/fixtures/basic.styl

```diff
@@ -1,2 +1,2 @@
 p
-  display: flex
```

Comment:
Since browserslists uses ` > 0.5%, last 2 versions, Firefox ESR, not dead` as its default I swapped flexbox out for a css rule that gets prefixes. https://github.com/browserslist/browserslist